@xen-orchestra/web/src/pages/host/[id]/network.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="host-network-view">
+    <div class="host-network-table-container">
+      <UiTitle type="h4" class="header">
+        <slot>{{ $t('pifs') }}</slot>
+        <template #actions>
+          <UiButton size="medium" variant="secondary" accent="info" :left-icon="faArrowsRotate">
+            {{ $t('scan-pifs') }}
+          </UiButton>
+        </template>
+      </UiTitle>
+      <div v-if="isReady" class="table-container">
+        <UiQuerySearchBar class="table-query" @search="(value: string) => (filter = value)" />
+        <UiTableActions>
+          <UiButton size="medium" variant="tertiary" accent="info" :left-icon="faEdit" disabled>
+            {{ $t('edit') }}
+          </UiButton>
+          <UiButton size="medium" variant="tertiary" accent="info" :left-icon="faTrash" disabled>
+            {{ $t('delete') }}
+          </UiButton>
+          <template #title>
+            <UiActionsTitle> {{ $t('table-actions') }}</UiActionsTitle>
+          </template>
+        </UiTableActions>
```

Comment:
Since the component `UiTableActions` already uses a `UiActionsTitle` in the `title` _slot_, and it accepts a `title` prop, you can simplify:
```suggestion
        <UiTableActions :title="$t('table-actions')">
          <UiButton size="medium" variant="tertiary" accent="info" :left-icon="faEdit" disabled>
            {{ $t('edit') }}
          </UiButton>
          <UiButton size="medium" variant="tertiary" accent="info" :left-icon="faTrash" disabled>
            {{ $t('delete') }}
          </UiButton>
        </UiTableActions>
```